phase1-coordinator/src/environment.rs

```diff
@@ -225,21 +224,17 @@ pub struct Environment {
     /// Returns the maximum duration a contributor can go without
     /// being seen by the coordinator before it will be dropped from
     /// the ceremony by the coordinator.
-    #[serde_as(as = "DurationSecondsWithFrac<String>")]
-    contributor_seen_timeout: chrono::Duration,
+    contributor_seen_timeout: time::Duration,
```

Comment:
Note: these won't be serialised in the same way when/if they are written to a file. It used to be something close to rfc3339 and now it'll be a unix timestamp. Seems like this isn't a concern but certainly worth pointing out.

Reply:
Yeah I already updated the coordinator json files to account for this. It broke some stuff but the fix was trivial :)